src/test/java/pages/SelectTimeSlotPage.java

```diff
@@ -3,11 +3,11 @@
 public class SelectTimeSlotPage extends commonMethods {
 
     public void selectTimeSlot() {
-        click("xpath","//*[@id='slots-month-August2023']/div[2]/div[1]/h3/button/span[5]/span/span[1]");
+        click("xpath","//*[@id='slots-month-October2023']/div[2]/div[1]/h3/button/span[5]/span");
```

Comment:
Need to make sure these don't need changing all the time.

Reply:
Yes agree. Looking out for options to change this. selenium is vigilant with this page object